rskj-core/src/test/java/co/rsk/remasc/RemascTestBlockStore.java

```diff
@@ -42,6 +45,7 @@
     Map<Long, Block> numberIndex = new HashMap<>();
     List<Block> blocks = new ArrayList<>();
     BigInteger totalDifficulty = ZERO;
+    Map<Long, List<IndexedBlockStore.BlockInfo>> index = new HashMap<>();
```

Comment:
Maybe this name could be improved, what do you think about something like `heightToInfos` or `infosByHeight`?